app/templates/buyers/brief_overview.html

```diff
@@ -15,10 +15,10 @@
 {% endblock %}
 
 {% block mainContent %}
-  <div class="grid-row">
+  <div class="govuk-grid-row">
     {% block before_heading %}
       {% if delete_requested %}
-        <div class="column-one-whole">
+        <div class="govuk-grid-column-one-whole">
```

Comment:
Looking at the [table of grid widths in govuk-frontend](https://github.com/alphagov/govuk-frontend/blob/v2.13.0/src/settings/_measurements.scss#L23-L30) I don't think `govuk-grid-column-one-whole` is included in govuk-frontend

Reply:
Ah interesting so it's actually doing nothing, eh?

Reply:
Yeah I think so

Reply:
Should be replaced with `govuk-grid-column-full`

Reply:
:+1: thanks